### PR TITLE
added word for love verb and infinitive prefix 

### PR DESCRIPTION
**Pull request by realblobii**
-zim for infinitives
and
nepettori for love

### CODE REVIEW

**Comment by realblobii** [comment]
*nepetori for love

File: Vocabulary/Suffixes.md [hunk 1]
```diff
@@ -36,6 +36,7 @@
 | -ku | establishes verb as uncertain/questioning | — |
 | -do | establishes verb as imperative | "should", "ought to" |
 | -kuka | establishes verb as politely questioning/requesting | "please" |
+| -zim | establishes verb as infinitive | "to ___" |
```
Comment (akpi816218) [inline Vocabulary/Suffixes.md:39]: @realblobii Shouldn't this be a temporal suffix? It actually modifies the tense of the verb by changing it to infinitive form...

**Review by akpi816218 — CHANGES_REQUESTED** [review]

File: Vocabulary/Suffixes.md (hunk 1 above)
Comment (realblobii) [reply Vocabulary/Suffixes.md:39]: yes my bad ill fix that quickly

**Comment by realblobii** [comment]
i fixed it @akpi816218

**Comment by realblobii** [comment]
just added word for egg as per a poll

**Review by akpi816218 — APPROVED** [review]

**Pull request merged**